Atomic/util.py

```diff
@@ -63,22 +63,61 @@ def get_docker_conf():
         dconf = c.info()
     return dconf
 
+
+def registries_tool_path():
+    if os.path.exists("/usr/libexec/registries"):
+        return "/usr/libexec/registries"
+    try:
+        return subprocess.check_output(['which', 'registries'])
+    except subprocess.CalledProcessError:
+        return None
+
+def load_registries_from_yaml():
+    # Returns in JSON
+    try:
+        return json.loads(check_output([registries_tool_path(), '-j']))
+    except subprocess.CalledProcessError:
+        return json.loads({})
+
 def get_registries():
     registries = []
-    dconf = get_docker_conf()
-    search_regs = [x['Name'] for x in dconf['Registries']]
-    rconf = dconf['RegistryConfig']['IndexConfigs']
-    # docker.io is special
-    if 'docker.io' in rconf:
-        registries.append({'hostname': 'registry-1.docker.io', 'name': 'docker.io', 'search': True, 'secure': True})
-        # remove docker.io
-        del(rconf['docker.io'])
-    for i in rconf:
-        search_bool = True if i in search_regs else False
-        registries.append({'hostname': i, 'name': i, 'search': search_bool, 'secure': rconf[i]['Secure'] })
+    if is_backend_available('docker'):
+        dconf = get_docker_conf()
+        search_regs = [x['Name'] for x in dconf['Registries']]
+        rconf = dconf['RegistryConfig']['IndexConfigs']
+        # docker.io is special
+        if 'docker.io' in rconf:
+            registries.append({'hostname': 'registry-1.docker.io', 'name': 'docker.io', 'search': True, 'secure': True})
+            # remove docker.io
+            del(rconf['docker.io'])
+        for i in rconf:
+            search_bool = True if i in search_regs else False
+            registries.append({'hostname': i, 'name': i, 'search': search_bool, 'secure': rconf[i]['Secure'] })
+    elif registries_tool_path() is not None:
```

Comment:
If we have registries.conf, we should remove this as the default.  Otherwise registries will never work.